charts/nexus-iq/values.yaml

```diff
@@ -155,6 +155,9 @@ configYaml:
   baseUrl: http://iq-server.demo
   sonatypeWork: /sonatype-work
   licenseFile: /etc/nexus-iq-license/license_lic
+  initialAdminPassword:
+  features:
+    enableUnauthenticatedPages:
```

Comment:
not specifying values will get it rendered into config.yml as:
```
features:
  enableUnauthenticatedPages: null
initialAdminPassword: null
```

Do you have good default values you could provide? Would you like to comment them away to be sample values available for those who want them?

Reply:
I see. These should be reasonable default values: ee84c495d5610923b549610d2c66ca36f87b0f91